pimd/pim_nht.c

```diff
@@ -898,9 +898,9 @@ int pim_ecmp_nexthop_lookup(struct pim_instance *pim,
 	uint32_t num_nbrs = 0;
 	char addr_str[PREFIX_STRLEN];
 
-	if (PIM_DEBUG_PIM_NHT) {
-		pim_inet4_dump("<addr?>", src->u.prefix4, addr_str,
-			       sizeof(addr_str));
+	pim_inet4_dump("<addr?>", src->u.prefix4, addr_str, sizeof(addr_str));
```

Comment:
Can we leave this guarded by debug flags? This seems unnecessary calling if no debug is enabled?